cmd/integration/integration.go

```diff
@@ -161,7 +161,7 @@ func startComponents(firstManifestURL, secondManifestURL, apiVersion string) (st
 
 	cl := client.NewOrDie(&client.Config{Host: apiServer.URL, Version: apiVersion})
 
-	helper, err := master.NewEtcdHelper(etcdClient, "")
+	helper, err := master.NewEtcdHelper(etcdClient, "", os.Getenv("ETCD_PREFIX"))
```

Comment:
Please move the checking of the env var into a separate testing package. So this call site should look like "etcdtest.Prefix()", which should call a Prefix() function in an etcdtest package, which should probably be in a subdirectory under tools.